src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const bodyDoc = document.querySelector('body');
+const mateLogo = document.querySelector('.logo');
```

Comment:
It's a good practice to check if the elements exist before performing operations on them. This can prevent potential errors if the elements are not found in the document. Consider adding an if-statement to check if `bodyDoc` and `mateLogo` are not null before using them.